third_party/rust/repos.bzl

```diff
@@ -24,3 +24,11 @@ def rust_repos(rules_rust = None, safe_ftdi = None):
         strip_prefix = "safe-ftdi-bazel-20220511_01",
         url = "https://github.com/lowRISC/safe-ftdi/archive/refs/tags/bazel-20220511_01.tar.gz",
     )
+
+    http_archive_or_local(
+        name = "serde_annotate",
+        local = serde_annotate,
+        sha256 = "5f7d1155d493ac6642de215ec5882e78f90f325de4bdcac94a9aa4c6121f0edc",
+        strip_prefix = "serde-annotate-bazel-0.0.1",
+        url = "https://github.com/cfrantz/serde-annotate/archive/refs/tags/bazel-0.0.1.tar.gz",
```

Comment:
Could we pin to a commit instead of a tag or perhaps as a secondary url? I don't like that pushing an update to this tag to that repository would break our fetch stage. I see we're already using tags in other places so don't let this block your work.

Reply:
Tags are supposed to be stable references (like everything else in git, they're actually mutable, but mutating a tag is ... counter-indicated by the operations manual).

I thought I had left a comment that this should not be merged until after https://github.com/lowRISC/serde-annotate/pull/6.  My plan for committing this is:
1. Get the bazel support in serde-annotate merged.
2. Perform a release of lowRISC/serde-annotate on github.
3. Update this repository reference to point to the release.
4. Submit.